internal/advert/postgres.go

```diff
@@ -153,6 +188,49 @@ func (repo PostgresAdvertRepository) Add(ctx context.Context, advert *advert.Adv
 	return trans.Commit()
 }
 
+func (repo PostgresAdvertRepository) GetList(ctx context.Context, langs []LanguageTag, limit int, offset int) ([]*advert.Advert, error) {
+	sqlExec := repo.db.WithContext(ctx)
+
+	var advertsDB []advertDB
+	_, err := sqlExec.Select(&advertsDB, "SELECT * FROM adverts LIMIT $1 OFFSET $2", limit, offset)
```

Comment:
LIMIT & OFFSET are kinda slow yea its not the scale to worry about but worth mentioning i think: https://ivopereira.net/efficient-pagination-dont-use-offset-limit

Reply:
Good comment, but sadly it is not possible with ID's equal to uuid, it would be possible with auto-incremented ids, anyway as you said I don't think it will be problem, we can always get better optimisation using cachine, especially in this case